src/PrestaShopBundle/Twig/Component/NavBar.php

```diff
@@ -28,14 +28,136 @@
 
 namespace PrestaShopBundle\Twig\Component;
 
+use Configuration;
+use Dispatcher;
+use PrestaShop\PrestaShop\Adapter\LegacyContext;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Routing\Exception\RouteNotFoundException;
 use Symfony\UX\TwigComponent\Attribute\AsTwigComponent;
+use Tab;
 
 #[AsTwigComponent(template: '@PrestaShop/Admin/Component/Layout/nav_bar.html.twig')]
 class NavBar
 {
-    public string $toggleNavigationUrl;
-    public string $defaultTabLink;
-    public string $psVersion;
-    public bool $collapseMenu;
-    public array $tabs;
+    public function __construct(
+        private readonly LegacyContext $context,
+        private readonly LoggerInterface $logger,
+    ) {
+    }
+
+    public function getToggleNavigationUrl(): string
+    {
+        return $this->context->getContext()->link->getAdminLink('AdminEmployees', true, [], [
```

Comment:
Or directly:

```suggestion
        return $this->context->getAdminLink('AdminEmployees', true, [], [
```

The LegacyContext has `getAdminLink` function.

Reply:
interesting, thanks also for the shortcut